audio/CMakeLists.txt

```diff
@@ -114,52 +127,86 @@ if(AUDIODRV_PULSE)
 	set(AUDIO_INCLUDES ${AUDIO_INCLUDES} ${PULSEAUDIO_INCLUDE_DIR})
 	# There seems to be no package or variable for pulse-simple.
 	set(AUDIO_LIBS ${AUDIO_LIBS} pulse-simple ${PULSEAUDIO_LIBRARY})
+	set(VGMAUDIO_PKGCONFIG_LDFLAGS ${VGMAUDIO_PKGCONFIG_LDFLAGS} "-lpulse-simple")
```

Comment:
I think this should be `-lpulse-simple -lpulse`. The latter is in the `PULSEAUDIO_LIBRARY` variabe.

Reply:
Done, just fixed+rebased+pushed